VenueDiscovery/Commands/ShowCountCommand.cs

```diff
@@ -56,18 +51,14 @@ public async Task HandleAsync(SlashCommandVeniInteractionContext c)
                     else if (venue.Location.DataCenter.Equals("Dynamis")) dynamisSum++;
                     else if (venue.Location.DataCenter.Equals("Chaos")) chaosSum++;
                     else if (venue.Location.DataCenter.Equals("Light")) lightSum++;
+                    else if (venue.Location.DataCenter.Equals("Materia")) materiaSum++;
```

Comment:
`/showcount` command wasn't showing number of venues in Materia. Kaeda found this, popped it in.